app/models/manageiq/providers/amazon/inventory/persister/target_collection.rb

```diff
@@ -24,6 +24,10 @@ def initialize_inventory_collections
 
   private
 
+  def references(collection)
```

Comment:
right, it's not really generic since it states :ems_ref, but maybe I could pack part of it to the TargetCollection